src/Core/Localization/Currency.php

```diff
@@ -166,7 +166,7 @@ public function getNumericIsoCode()
      */
     public function getSymbol($localeCode)
     {
-        if (!isset($this->symbols[$localeCode])) {
+        if (!array_key_exists($localeCode, $this->symbols)) {
```

Comment:
I believe `isset` is faster no?

Reply:
@tomlev what's the reason of this change?

Reply:
Maybe he needs to check for null value

Reply:
exactly!

Reply:
some locales are getting "null" symbols, which must be considered as existant ones